recipe/meta.yaml

```diff
@@ -26,19 +26,27 @@ source:
     sha256: {{ sha256 }}
     patches:
       - support_new_cuda_layout.diff  # [(linux and cuda_compiler_version == "12.0") or (build_platform != target_platform)]
+      - support_nvtx3_win.diff        # [win64]
 
 build:
-  number: 3
+  number: 4
   # TODO: turn on win64 + CUDA 12 once it's ready
   # TODO: turn on aarch64 + CUDA 12 once glibc 2.28 is ready
   skip: true  # [cuda_compiler_version == "None" or (cuda_compiler_version == "12.0" and not linux64)]
   script:
+    # When cross compiling we hit OOM (including "no space on device" in /tmp) very easily
+    - mkdir -p $SRC_DIR/tmp                                                      # [build_platform != target_platform]
+    - export TMPDIR=$SRC_DIR/tmp                                                 # [build_platform != target_platform]
+    - export TMP=$TMPDIR                                                         # [build_platform != target_platform]
+    - export TEMP=$TMPDIR                                                        # [build_platform != target_platform]  
+    # To reduce the chance of hitting OOM
     - export CUPY_NVCC_GENERATE_CODE="arch=compute_60,code=sm_60;arch=compute_70,code=sm_70;arch=compute_80,code=sm_80"  # [aarch64 or ppc64le]
     # CuPy default detects CUDA from nvcc, but on Conda-Forge's dockers nvcc lives in a different place...
     # With conda-forge/nvcc-feedstock#58, CUDA_PATH is set correctly
-    - export NVCC=$(which nvcc)                                                  # [linux]
-    {% if cuda_major_minor == (12, 0) %}
-    - export CUPY_NUM_BUILD_JOBS=1                                               # [aarch64 and (build_platform == target_platform)]
+    - export NVCC="$(which nvcc)"                                                # [linux and cuda_compiler_version == "10.2"]
+    - export NVCC="$(which nvcc) -objtemp"                                       # [linux and cuda_compiler_version != "10.2"]
+    - export CUPY_NUM_BUILD_JOBS=1                                               # [aarch64 or ppc64le]
```

Comment:
Do we still need this when going back to QEMU?

Reply:
By eyeballing [the latest run](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=719383&view=results) with the two runs ([1](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=719010&view=results), [2](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=718878&view=results)) from yesterday, I think setting `CUPY_NUM_BUILD_JOBS=1` hurts slightly, let's revert.